### PR TITLE
remove temporary exemption

### PR DESCRIPTION
**Pull request by davidbielik**
android 12 now provides permanent exemption instead of temporary exemption for upgraded users.

### CODE REVIEW

File: _docs/_developer_guide/platform_integration_guides/android/android_13.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Android 13 introduces a [major change][3] in how users manage apps that send pus
 
 This new permission follows a similar pattern to iOS and Web push, where you only have one attempt to obtain permission. If a user chooses `Don't Allow` or dismisses the prompt, your app cannot ask for permission again.
 
-Note that apps are granted a [temporary exemption][4] intended for users who previously had push notifications for that app enabled prior to updating to Android 13. These users will remain eligible to receive push from the app until (A) they explicitly decline the permission prompt when it's shown (or within system settings), or (B) the app is opened for the first time after a user upgrades to Android 13.
+Note that apps are granted an [exemption][4] for users who previously had push notifications enabled prior to updating to Android 13. These users [will remain eligible][8] to receive push when they update to Android 13 without requiring permission.
```
Comment (bryanlogan) [inline _docs/_developer_guide/platform_integration_guides/android/android_13.md:38]: Pedantic nit: "without requiring to request permission".  Saying "without requiring permission" implies you don't have permission.  My guess if they have the permission granted.

Comment (davidbielik) [reply _docs/_developer_guide/platform_integration_guides/android/android_13.md:38]:
```suggestion
Note that apps are granted an [exemption][4] for users who previously had push notifications enabled prior to updating to Android 13. These users [will remain eligible][8] to receive push when they update to Android 13 without having to request permission.
```

**Comment by davidbielik** [comment]
@KellieHawks any chance this could be released today? if not no worries

**Review by bryanlogan — APPROVED** [review]

**Review by KellieHawks — APPROVED** [review]

**Comment by KellieHawks** [comment]
@davidbielik, happy to get this out today. 👍🏻

**Pull request merged**